recipes/pyretis/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pyretis" %}
+{% set version = "0.9.2.dev2" %}
```

Comment:
after I did the previous suggestions indicated, the tests  
ci/circleci 
continuous-integration/appveyor/pr
continuous-integration/travis-ci/pr

failed